pages/api/graphql.ts

```diff
@@ -16,7 +19,11 @@ const apolloServer = new ApolloServer({
   /* Syncs server schema (used for server static generation) and api route server settings. 
   By default apolloServer accepts uploads, while schema-generated server does not.*/
   uploads: false,
-  plugins: [apolloLogPlugin]
+  plugins: [apolloLogPlugin],
+  ...(isPreview && {
```

Comment:
The playground still work locally because when `VERCEL_ENV` is undefined, `ApolloServer` will handle enabling/disabling the playground based on the `NODE_ENV` variable.